blazor-toc.html

```diff
@@ -1497,6 +1497,15 @@
 					<li>
 						<a href="/blazor/datagrid/how-to/css-isolation-for-grid">CSS Isolation for DataGrid</a>
 					</li>
+					<li>
+						<a href="/blazor/datagrid/how-to/hide-checkbox-in-column-header">Hide CheckBox in Column Header</a>
```

Comment:
move this topic under checkbox sub topics. next to https://blazor.syncfusion.com/documentation/datagrid/selection#checkbox-selection-mode